scripts/run_e2e_tests.sh

```diff
@@ -107,6 +107,29 @@ if [ -d "../protractor-screenshots" ]; then
   rm -r ../protractor-screenshots
 fi
 
+# Pass different suites for tests.
```

Comment:
Should add documentation at top of file explaining how to use this.

(Also, just to check, you've tested manually, right?)